Rubberduck.Parsing/Annotations/ITestAnnotation.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Rubberduck.Parsing.Annotations
+{
+    public interface ITestAnnotation
```

Comment:
I would want this to inherit from `IAnnotation`, to possibly allow the compiler to optimize some things. It also better matches the semantics of what's intended

Reply:
For whatever reason I didn't think of that!